src/nls/pl/strings.js

```diff
@@ -129,6 +129,7 @@ define({
     "SAVE_CLOSE_MULTI_MESSAGE"          : "Czy chcesz zapisać zmiany w następujących plikach?",
     "EXT_MODIFIED_TITLE"                : "Zmiany zewnętrzne",
     "CONFIRM_DELETE_TITLE"              : "Potwierdź usunięcie",
+    "CONFIRM_FILE_DELETE"               : "Czy na pewno chcesz usunąć plik <span class='dialog-filename'>{0}</span>?",
     "CONFIRM_FOLDER_DELETE"             : "Czy na pewno chcesz usunąć folder <span class='dialog-filename'>{0}</span>?",
```

Comment:
I think "Czy na pewno usunąć" will look better both for file and folder texts. 
"chcesz" is unnecessary extender.